test/src/test/java/lib/form/SecretTextareaTest.java

```diff
@@ -144,7 +144,7 @@ private static TestBuilder fromStringWithDescription(String secret, String descr
 
         @DataBoundConstructor
         public TestBuilder(Secret secret) {
-            this.secret = secret;
+            this.secret = fixEmptySecret(secret);
```

Comment:
The test was checking using ` j.assertEqualDataBoundBeans` on the `TestBuilder`s and the new Secret was set to the empty String.
This adapts the TestBuilder to convert an empty secret to null, as it was less intrusive and also the test would still cehck the description.